### PR TITLE
Add corresponding author stats

### PR DESCRIPTION
**Pull request by dhimmel**
from https://github.com/greenelab/iscb-diversity/commit/3f0415686c4d59f423852679258984c63754e038

### CODE REVIEW

File: content/10.methods.md [hunk 1]
```diff
@@ -69,12 +69,16 @@ We were able to retrieve authorship information for all but 6 of these articles
 
 To determine corresponding authors for an article, we relied on PMC data if available (20,696 articles) and otherwise fellback to PubMed data (9,053 articles).
 Almost all articles without PMC data were from _Bioinformatics_, since its a "selective deposit" rather than "full paricipation" journal in PMC.
-We assumed the last author was corresponding when coded corresponding author status was not available (120 articles from PMC and all articles from PubMed).
 
 We performed further analysis on PMC authors to learn more about corresponding author practices.
+First, we evaluated how to infer a corresponding author when coded corresponding status was not available.
```
Comment (trangdata) [inline content/10.methods.md]:
```suggestion
First, we developed and evaluated a method to infer a corresponding author when the coded corresponding status was not available.
```

File: content/10.methods.md [hunk 2]
```diff
@@ -69,12 +69,16 @@ We were able to retrieve authorship information for all but 6 of these articles
...
+For papers with multiple authors and at least one corresponding author, the first author was corresponding 43% of the time, whearas the last author was corresponding 62% of the time.
+Therefore, we assumed the last author was corresponding when coded corresponding author status was not available (120 articles from PMC and all articles from PubMed).
+
+Second, we investigated the number of corresponding authors for PMC articles.
 81% of articles had a single corresponding author.
```
Comment (trangdata) [inline content/10.methods.md]:
```suggestion
81% of these articles had a single corresponding author.
```

**Review by trangdata — COMMENTED** [review]
I suggested some rephrasing.

**Review by cgreene — APPROVED** [review]
If @trang1618's changes are made I approve.

**Pull request merged**